lib/MusicBrainz/Server/Data/Utils.pm

```diff
@@ -256,9 +258,9 @@ sub get_area_containment_query {
         SELECT ad.descendant, ad.parent, ad.depth
           FROM area_descendants ad
           JOIN area a ON a.id = ad.parent
```

Comment:
I think we could also move the JOIN itself to the `$levels_condition` (don't seem to use `a` otherwise)?

Reply:
Sure, seems reasonable. Done!